bin/disco_vpc_ui.py

```diff
@@ -84,17 +102,31 @@ def destroy_vpc_command(args):
         sys.exit(2)
 
 
+def update_vpc_command(args):
+    """ handle vpc update command actions"""
+    if args.vpc_name:
+        vpc = DiscoVPC.fetch_environment(environment_name=args.vpc_name)
+    else:
+        vpc = DiscoVPC.fetch_environment(vpc_id=args.vpc_id)
+
+    if vpc:
+        vpc.update(args.dry_run)
+    else:
+        logging.error("No matching VPC found")
+        sys.exit(2)
+
+
 def list_vpc_command(args):
     """ handle list vpcs command actions """
     for vpc_env in DiscoVPC.list_vpcs():
-        line = u"{0}\t{1:<15}".format(vpc_env.id, vpc_env.tags.get("Name", "-"))
+        line = u"{0}\t{1:<15}".format(vpc_env['id'], vpc_env['tags'].get("Name", "-"))
```

Comment:
Would it be a bad idea to have **default** as default name for VPCs missing a name? Since only the default VPC created by Amazon will be the one without a name, name **default** would be a good fit.